package.json

```diff
@@ -27,11 +27,14 @@
     "got": "^6.7.1",
     "gravatar-url": "^1.1.0",
     "lodash": "^4.17.4",
+    "marked": "^0.3.6",
     "ms": "^0.7.2",
     "nice-package": "^3.0.1",
     "numeral": "^2.0.4",
     "pouchdb-http": "^6.0.2",
-    "traverse": "^0.6.6"
+    "traverse": "^0.6.6",
+    "url": "https://github.com/webcomponents/URL",
```

Comment:
Seems like you can use url-polyfill: https://github.com/webcomponents/URL/blob/812ca9b2baae653290aceb36362d219cb96c0f35/package.json

I try to avoid dependencies as http/s/git, it's usually slow to install and has no explicit versioning.